persistence-modules/java-mongodb/README.md

```diff
@@ -1,12 +1,13 @@
-## MongoDB
-
-This module contains articles about MongoDB in Java. 
-
-### Relevant articles:
-
-- [A Guide to MongoDB with Java](http://www.baeldung.com/java-mongodb)
-- [A Simple Tagging Implementation with MongoDB](http://www.baeldung.com/mongodb-tagging)
-- [MongoDB BSON Guide](https://www.baeldung.com/mongodb-bson)
-- [Geospatial Support in MongoDB](https://www.baeldung.com/mongodb-geospatial-support)
-- [Introduction to Morphia – Java ODM for MongoDB](https://www.baeldung.com/mongodb-morphia)
-- [MongoDB Aggregations Using Java](https://www.baeldung.com/java-mongodb-aggregations)
+## MongoDB
+
+This module contains articles about MongoDB in Java. 
+
+### Relevant articles:
+
+- [A Guide to MongoDB with Java](http://www.baeldung.com/java-mongodb)
+- [A Simple Tagging Implementation with MongoDB](http://www.baeldung.com/mongodb-tagging)
+- [MongoDB BSON Guide](https://www.baeldung.com/mongodb-bson)
+- [Geospatial Support in MongoDB](https://www.baeldung.com/mongodb-geospatial-support)
+- [Introduction to Morphia – Java ODM for MongoDB](https://www.baeldung.com/mongodb-morphia)
+- [MongoDB Aggregations Using Java](https://www.baeldung.com/java-mongodb-aggregations)
+- [MongoDB BSON to JSON](https://www.baeldung.com/bson-to-json)

```

Comment:
Let's remove that for now since we don't want a dead link to be present there before we actually publish
```suggestion
```